src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
i like them in a separate file because i know exactly where to go if i want to modify them, the RTL stylesheet. rather than skimming through the whole file to find bits and pieces of RTL.